mockup-users.json

```diff
@@ -1,46 +1,52 @@
 {
-  "users": [
+  "User": [
```

Comment:
this object is named User but it is actually a list of users? why did you choose to change this?

Reply:
also try using decent data in your mocks

Reply:
I tested the http request from my client using json-server (before trying with the real api) and from what i've seen it expects as key the endpoint (that is why I switched it to 'User', to keep it the same way I found it on the server). I didn't dive deeper into details regarding this tool, the reason was to see if the route behaves as expected.